public/index.html

```diff
@@ -5,27 +5,59 @@
   <meta charset="utf-8">
   <title>Application Landscape Viz</title>
   <link rel="stylesheet" href="css/style.css">
-  <script src="libs/gojs-lib/go-debug.js"></script>
+
+  <script src="../gojs-lib/go-debug.js"></script>
```

Comment:
That does work. The client cannot exit the public directory. Please locate all library files (eg gojs and go-debug) inside public/libs/...

Reply:
This is what it should look like: 
```html
<script src="libs/gojs-lib/go-debug.js"></script>
```